CMakeLists.txt

```diff
@@ -136,8 +136,8 @@ install (DIRECTORY ${PROJECT_SOURCE_DIR}/src/quartz/
         DESTINATION include/quartz
         FILES_MATCHING PATTERN "*.h")
 
-add_subdirectory(src/test)
-add_subdirectory(src/benchmark)
+# add_subdirectory(src/test)
+# add_subdirectory(src/benchmark)
```

Comment:
It seems that this is disabling all tests and benchmarks. Could you please revert this?

Reply:
oh I forgot to comment back...